spdx/parsers/parse_anything.py

```diff
@@ -21,10 +21,13 @@
 from spdx.parsers.builderexceptions import FileTypeError
 
 
-def parse_file(fn):
+def parse_file(fn, encoding="utf-8"):
+    in_mode = "r"
     builder_module = jsonyamlxmlbuilders
     read_data = False
     if fn.endswith(".rdf") or fn.endswith(".rdf.xml"):
+        in_mode = "rb"
```

Comment:
I don't think we need this mode as a graph can be parsed from a file-like object.